main.go

```diff
@@ -43,6 +47,7 @@ func init() {
 	flag.Var(&metricAddr, "m", "Metrics addr")
 	flag.IntVar(&gomaxprocs, "maxprocs", 0, "set GOMAXPROCS variable to limit cpu")
 	flag.StringVar(&pcapPath, "pcap", "", "path to PCAP file")
+	flag.BoolVar(&exitWithParent, "exit-with-parent", false, "Exit with parent process")
```

Comment:
Should be: "Kill slirpnetstack with SIGTERM when the parent process dies"

Reply:
ok, but you already merged it :)